EGXCOLIM/EGXCOLIM-IOC-02App/Db/Makefile

```diff
@@ -0,0 +1,18 @@
+TOP=../..
```

Comment:
Second IOC doesn't make sense for this device as it's an lvdcom of a single underlying vi